src/content/en/fundamentals/_project.yaml

```diff
@@ -33,4 +33,4 @@ announcement:
         vertical-align: middle;
         margin-right: 8px;
       }
-    </style> The <a href="https://developer.chrome.com/devsummit/?utm_source=webfu&utm_medium=hpp-banner&utm_campaign=chromedevsummit_may18" class="gc-analytics-event" data-category="webFu" data-label="cds18-hpp" data-action="banner">#ChromeDevSummit</a> site is live, happening Nov 12-13 in San Francisco, CA<br> Check it out for details and request an invite. We'll be diving deep into modern web tech & looking ahead to the platform's future.
+    </style> The <b>Chrome Dev Summit</b> begins next Monday at 10:00am (Pacific). Don’t forget to tune into the livestream on the Google Chrome Developers channel on YouTube to follow the event. <a href="https://developer.chrome.com/devsummit/schedule" class="gc-analytics-event" data-category="webFu" data-label="cds-banner-schedule">#ChromeDevSummit Schedule.</a>
```

Comment:
Hmm. I wonder if these sneaked in during a `git pull` right . before your changes were submitted. Could we try to ignore _project.yaml changes related to CDS here locally?

Reply:
Would rebasing from master fix it as well?